docs/conf/modules.example.conf

```diff
@@ -2110,7 +2110,7 @@
 
 # You must define <sasl:target> to the name of your services server so
 # that InspIRCd knows where to send SASL authentication messages and
-# when it should enable the SASL capability.
+# when it should enable the SASL capability.
```

Comment:
Interesting that GitHub shows the character as a normal space.  It's not.  It's this thing:
```
<| >  <M- >  160,  Hex a0,  Oct 240, Digr NS
```
A regular space is:
```
< >  32,  Hex 20,  Oct 040, Digr SP
```